### PR TITLE
OOC prefix proc

### PR DESCRIPTION
**Pull request by realforest2001**
# About the pull request
Adds a proc to check OOC prefixes, and adds prefixes for Staff and Mentor. Allows for clear indication of who's talking in OOC, and useful for planned restructure.

# Explain why it's good for the game

# Testing Photographs and Procedure

<details>
<summary>Screenshots & Videos</summary>

Put screenshots and videos here with an empty line between the screenshots and the `<details>` tags.

</details>

# Changelog

:cl:
add: Added prefixes in OOC for Staff and Mentors. Created a proc to check for these prefixes and Donator.
fix: Fixed isCouncil check.
/:cl:

### CODE REVIEW

File: code/game/verbs/ooc.dm
Comment (Zonespace27) [inline code/game/verbs/ooc.dm:79]: codedoc would be nice

**Review by nauticall — CHANGES_REQUESTED** [review]
gave u better icons to you via DM, let me know of any suggestions to make. 

putting on hold until some version of better stuff is implemented ✋

**Comment by realforest2001** [comment]
Closing for now as I'm dealing with medical issues that are taking my attention. Will reopen when I have the abilty to do it justice.

**Comment by cm13-github** [comment]
Conflicts have been resolved. A maintainer will review the pull request shortly.

**Comment by morrowwolf** [comment]
Everyone has 10 trillion flags

**Comment by morrowwolf** [comment]
image

File: code/game/verbs/ooc.dm
Comment (Segrain) [inline code/game/verbs/ooc.dm:70]: All of this should go into `handle_ooc_prefix()`. Otherwise you are adding membership icon and flag once _per everybody listening_.
image

**Review by harryob — APPROVED** [review]

**Pull request merged**
